scrapy/pqueues.py

```diff
@@ -63,21 +52,36 @@ def __init__(self, crawler, downstream_queue_cls, key, startprios=()):
         self.curprio = None
         self.init_prios(startprios)
 
+    @classmethod
+    def bring_prios_up_to_date(_, startprios):
```

Comment:
Are you sure you want to make this method public? It may be better to prefix it with `_` and not need to worry about keeping it backward-compatible in the future.